tests/e2e/helpers.ts

```diff
@@ -36,14 +36,18 @@ export async function logOut({ page }) {
   await page.goto("/");
 }
 
-/* Recursive function to click any archive buttons that appear on screen. */
 async function archiveReports(buttons: Locator, page) {
   const archiveButtons = await buttons.all();
-
   if (archiveButtons.length > 0) {
+    const archivePromise = page.waitForResponse(
+      (response) =>
+        response.url().includes("/reports/archive/WP/PR/") &&
+        response.status() == 200
+    );
+
     await archiveButtons[0].click();
-    await page.waitForResponse("**/reports/archive/WP/PR/**");
-    await archiveReports(buttons, page);
+    await archivePromise;
+    await archiveReports(buttons);
```

Comment:
`archiveReports` is expecting two arguments.

Reply:
Thanks for this catch!